nodejs/docs/sequelize/specific_details.md

```diff
@@ -0,0 +1,72 @@
+# Sequelize specific details
+
+## Validations in bulk methods
+
+Suppose you have a User model in Sequelize and you have a validation inside.
+
+```javascript
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define(
+    'user',
+    {
+      id: { type: DataTypes.INTEGER, allowNull: false, autoIncrement: true, primaryKey: true },
+      userName: { type: DataTypes.STRING, allowNull: false },
+      userType: { type: DataTypes.STRING, allowNull: false, validate: { isIn: [['regular', 'administrator']] }
+    },
+    {}
+  );
+
+  User.associate = function(models) {};
+
+  return User;
+};
+```
+
+If you create an user in the "normal" way and you do not respect the validation, an error will be thrown.
+For example:
+
+```javascript
+return User.create(
+  {
+    userName: 'wolox',
+    userType: 'notAValidOne'
+  }
+);
+```
+throws an error.
+
+
+But if you handle users with a `bulk` method, such as `bulkCreate`, `bulkUpdate`, or another one, the validation `won't trigger` by default.
+
+```javascript
+return User.bulkCreate([
+  {
+    userName: 'wolox1',
+    userType: 'notAValidOne'
+  },
+  {
+    userName: 'wolox2',
+    userType: 'notAValidOne'
+  }  
+]);
+```
+
+This will be fine and your instances will be created successfully.
```

Comment:
I would say something like _"Although there are invalid values, the instances are created successfully, due the behavior of bulk actions in Sequelize."_